index.css

```diff
@@ -139,7 +146,7 @@ section ul.links > li {
 	-webkit-align-self: auto;
 	-ms-flex-item-align: auto;
 	align-self: auto;
-	margin: 0.5em;
+	margin: 0.4em;
 }
```

Comment:
Why?

Reply:
So that the flex doesn't wraps itself  to some other arrangements of buttons on less screen width.

Reply:
This seems really fragile. Atop of that it doesn't work at all. See the screenshots I posted. The GitHub button is in the first row even though it was in the second before your changes.